src/Database/LSMTree/Internal/Run/Construction.hs

```diff
@@ -241,7 +186,7 @@ paEmpty rangeFinderPrecision = PageAcc {
 
 paAddElem ::
      SerialisedKey
-  -> Entry RawValue BlobRef
+  -> Entry SerialisedValue (Word64, Word32)
```

Comment:
Yes, there was some tension between the BlobRef type and the tuples (e.g. returned when reading from a RawPage). I think going with the tuple for now is okay (although maybe there could be type synonym for it), but in the long run it would make sense to use a dedicated type I think.

Reply:
Makes sense -- until then, we'll use the tuple